test_manage_arkime/core/test_usage_report.py

```diff
@@ -30,21 +30,21 @@ def test_WHEN_UsageReport_get_report_THEN_as_expected():
     # Check the results
     expected_report = (
```

Comment:
Can we please add either a new test or update a line in the existing test to show the difference in behavior when we have `None` as the old value and when we don't?